front/app/components/VoteControl/VotingDisabled.tsx

```diff
@@ -74,78 +75,70 @@ const VotingDisabled = memo(({ projectId, votingDescriptor }: Props) => {
   const project = useProject({ projectId });
   const authUser = useAuthUser();
 
-  const onVerify = (event) => {
+  const onVerify = (
+    pcType: IParticipationContextType,
```

Comment:
Enforced clearer types as an argument instead of doing a isNilOrError check of project inside this function. We now have that isNilOrError check before the component that has the onVerify as an onClick is rendered.

Reply:
Nice 👍